clang/test/Driver/sycl-device-lib.cpp

```diff
@@ -105,6 +105,8 @@
 /// test llvm-link behavior for linking device libraries
 // RUN: %clangxx -fsycl %s --sysroot=%S/Inputs/SYCL -### 2>&1 \
 // RUN:   | FileCheck %s -check-prefix=SYCL_LLVM_LINK_DEVICE_LIB
+// RUN: %clangxx -fsycl -save-temps %s --sysroot=%S/Inputs/SYCL -### 2>&1 \
+// RUN:   | FileCheck %s -check-prefix=SYCL_LLVM_LINK_DEVICE_LIB
```

Comment:
I'd suggest a similar `sycl-device-lib-win.cpp` update for safeguarding purposes.